pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -385,8 +390,9 @@ func NewOperatorDeployment(namespace string, repository string, imagePrefix stri
 	kubeVirtVersionEnv string, virtApiShaEnv string, virtControllerShaEnv string,
 	virtHandlerShaEnv string, virtLauncherShaEnv string) (*appsv1.Deployment, error) {
 
-	podAntiAffinity := newPodAntiAffinity("kubevirt.io", "kubernetes.io/hostname", metav1.LabelSelectorOpIn, []string{"virt-operator"})
-	name := "virt-operator"
+	const virtOperator = "virt-operator"
+	podAntiAffinity := newPodAntiAffinity(kubevirtLabelKey, kubernetesHostnameTopologyKey, metav1.LabelSelectorOpIn, []string{virtOperator})
+	name := virtOperator
```

Comment:
i don't think you need this name variable anymore. just use virtOperator to replace where name is used